docs/en/sdk-manual/web.md

```diff
@@ -0,0 +1,236 @@
+| _search            | each time a user performs a site search, indicated by the presence of a URL query parameter, by default we detect `q`, `s`, `search`, `query` and `keyword` in query parameters.  | 1._search_key (the keyword name)<br>2._search_term (the search content)                                                                                                                                                                                                                                             |
+| _click             | each time a user clicks a link that leads away from the current domain (or configured domain list)                                                                                | 1._link_classes (the content of `class` in tag `<a>` )<br>2._link_domain (the domain of `herf` in tag `<a>` )<br>3._link_id (the content of `id` in tag `<a>` )<br>4._link_url (the content of `herf` in tag `<a>` )<br>5._outbound (if the domain is not in configured domain list, the attribute value is `true`) |
+| _clickstream_error | event_name is invalid or user attribute is invalid                                                                                                                                | 1._error_code <br>2._error_value                                                                                                                                                                                                                                                                                    |
+
+#### Session definition
+
+In Clickstream Web SDK, we do not limit the total time of a session, as long as the time between the next entry of the browser and the last exit time is within the allowable timeout period, we consider the current session to be continuous.
+
+**_session_start**: When the website open for the first time, or the browser was open to the foreground and the time between the last exit exceeded `session_time_out` period.
+
+**_session_duration**: We calculate the `_session_duration` by minus the current event create timestamp and the session's `_session_start_timestamp`, this attribute will be added in every event during the session.
+
+**session_time_out**: By default is 30 minutes, which can be customized through the configuration api.
+
+**_session_number**: The total number of session distinct by session id, and `_session_number` will be appeared in every event's attribute object.
+
+#### 4.1.2 User engagement definition
```

Comment:
Why does this section only contain a number bullet?

Reply:
This was copied from the RFC document, deleted.